team.go

```diff
@@ -84,8 +91,8 @@ func (c *Client) RemoveUserFromTeam(teamID, userID string) error {
 }
 
 // AddUserToTeam adds a user to a team.
-func (c *Client) AddUserToTeam(teamID, userID string) error {
-	_, err := c.put("/teams/"+teamID+"/users/"+userID, nil, nil)
+func (c *Client) AddUserToTeam(options AddUserToTeamOptions) error {
```

Comment:
This modification makes this a breaking change, which will add some delays in merging.

How would you feel about creating a new method, and update `AddUserToTeam()` to call that one and indicate the other should be used instead? Then we can file an issue to clean this up in `v2.0.0`.